### PR TITLE
Product listing page

### PR DESCRIPTION
**Pull request by SRUJANA-PENUGONDA13**
### Added Product Listing Page
product-listing

### CODE REVIEW

File: src/product-listing.html [hunk 1]
```diff
@@ -0,0 +1,372 @@
...
+              <button class="heart-btn">
+                <i class="fas fa-heart fa-2x"></i>
+              </button>
+            </div>
+            <div class="card-content">
+              <h2 class="item-name">Traditional Dress</h2>
+              <!-- <p class="item-desc">
+                boAt Xtend Smartwatch with Alexa Built-in, 1.69”, Health Monitor
+                and 14 sports mode.
+              </p> -->
```
Comment (sanjay-xdr) [inline src/product-listing.html]: remove this comment if it is unwanted

**Review by sanjay-xdr — COMMENTED** [review]
everything looks good

**Review by Rajitha04 — CHANGES_REQUESTED** [review]
Screenshot_2022-02-20-13-27-45-20_40deb401b9ffe8e1df2f1cc5ba480b12
 In the filters, 2 stars and above is out of the padding area, try to alter the padding rest looks good!

**Comment by SRUJANA-PENUGONDA13** [comment]
Updated suggested changes.
Thank you @sanjay-xdr @Rajitha04

**Review by Rajitha04 — APPROVED** [review]
Looks good! U can merge

**Pull request merged**